frontend/src/components/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -32,13 +34,28 @@ type ReviewRow = {
 
 const AdminDashboard = (): React.ReactElement => {
   const [data, setData] = useState<ReviewResponse[]>([]);
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
+  const [reviewName, setReviewName] = useState("");
```

Comment:
Could you make these names more descriptive as well? ie explicitly state that this is the review to be deleted